spec/share/extensions/foo-include.js

```diff
@@ -0,0 +1,13 @@
+Opal.Asciidoctor.$$scope.Extensions.register(function() {
+  this.includeProcessor(function() {
+    var self = this;
+    self.handles(function (target) {
+      return target.endsWith('.foo');
```

Comment:
We already test the include macro (ie. if `handles` always returns `true`, the tests suite is failing) but you are right, I will add an explicit test next to the "foo-include-ex" test.